content/post/2022-08-06-netflix-description-analysis/index.Rmd

```diff
@@ -165,10 +165,345 @@ netflix_films  %>%
   scale_y_percent(n.breaks = 8) +
   labs(fill = NULL, x = NULL, y = "Percentage of film",
        title = "Number of film per category",
-       subtitle  = paste0("Shannon index: ", shannon_index_netflix)) + 
+       subtitle  = paste0(" Shannon Equitability Index: ", shannon_index_netflix)) + 
   theme(legend.position = "bottom",
         axis.ticks.y = element_blank(),
         axis.text.y = element_blank(),
         text = element_text(size = 18)) +
   coord_flip()  
 ```
+
+According to the **Shannon Equitability Index** the dataset is balanced. For this reason we can go on with our analysis.
+
+Another interesting thing that we could be interesting in is counting the number of words by category this because a word that is recurrent in a category will be very useful in the prediction.
```

Comment:
sostutisici interesting con useful o impressive o qualcosa senno è ripetitivo
toglierei il this